### DISCUSSION TITLE
Stopping navigation is way too convoluted

### DISCUSSION THREAD

**Discussion by guillaumeboehm** [osmandapp/OsmAnd#15462] [General]
I started using osmand recently and I really like it but I always find myself struggling to turn navigation off.
Most of the discomfort is when I'm driving, let's say I'm coming up to my home and I don't need navigation anymore, the current solution to turn navigation off is, I think, way too convoluted and I'm struggling a lot to turn it off. But even when not driving it can be quite annoying.

I picture very well a `cross` button in the little button cluster on the bottom left of the screen, that is auto-hidden during navigation.

I understand that a one tap button to stop the navigation could be annoying for misstypes or such, but maybe this could be an accessibility setting?

**Comment by vshcherb** [comment]
Created an issue to follow up https://github.com/osmandapp/OsmAnd/issues/15479
